tests/js/spec/components/createProject.spec.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import {shallow, mountWithTheme} from 'sentry-test/enzyme';
 
-import {CreateProject} from 'app/components/createProject';
+import {CreateProject} from 'app/views/projectInstall/createProject';
```

Comment:
Can we move this test file too so it matches the component?

Reply:
Done.

Reply:
done.